dnf/cli/commands/__init__.py

```diff
@@ -106,6 +106,7 @@ class Command(object):
     def __init__(self, cli):
         # :api
         self.cli = cli
+        self.forms = None
```

Comment:
I would remove self.forms from configure of install/remove commands and put them into the particular run methods as a local variables. Marking this as a api does not make any sense to me.